deploy/sre-prometheus/ocm-agent/100-ocm-agent-node-condition.PrometheusRule.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-node-condition-managed-notification-alerts
+    role: alert-rules 
+  name: sre-node-condition-managed-notification-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-node-condition-managed-notification-alerts.record
+    rules:
+    - expr: label_replace
+            (
+                clamp(mapi_machine_created_timestamp_seconds, 0, 1) 
+                * on (node) group_right(name)
+                kube_node_status_condition{status!="false"}>0,
+                "machineset", 
+                "$1", 
+                "name", 
+                "^(.*)-[^-]*$"
+            )
+      record: sre:node:condition
+
+  # available conditions: https://kubernetes.io/docs/reference/node/node-status/#condition 
+  # TODO convert to single alert when OCM Agent supports templatized messages: https://issues.redhat.com//browse/OSD-14130
+  - name: sre-node-condition-managed-notification-alerts.alert
+    rules:
+    - alert: NodeConditionDiskPressureNotificationSRE
+      expr: count by (machineset, condition, status)
+            (
+              count by(node) (cluster:cpu_core_node_labels{label_node_role_kubernetes_io_infra="",label_node_role_kubernetes_io_master=""})
```

Comment:
Will this actually exclude infra nodes?  In my testing for other alerts I had to explicitly exclude infra nodes from the list of worker nodes since they will appear with both labels, however I was accomplishing this a different way so I'm not sure if that applies here. See: https://github.com/openshift/managed-cluster-config/blob/master/deploy/sre-prometheus/ocm-agent/100-ocm-agent.PrometheusRule.yaml#L43-L60

Reply:
Yea, tested and verified it does exclude infra.  The label `label_node_role_kubernetes_io_infra` is set to `"true"` when that node has the role `infra`.

Reply:
Awesome, thanks for confirming :)